### PR TITLE
chore: add changeset for webhook features

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

add changeset for webhook features

## Testing

N/A

## Checklist

- [x] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [ ] This PR is not applicable for the checklist

### CODE REVIEW

File: .changeset/pink-eggs-divide.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  
+## Updates
+
+- schemas: add `name`, `events`, `signingKey`, and `enabled` fields to the `hook` schema
+- core: change the `user-agent` value from `Logto (https://logto.io)` to `Logto (https://logto.io/)` in the webhook request headers
+- core: deprecate `event` field in all hook-related APIs, use `events` instead
+- core: deprecate `retries` field in the `HookConfig` for all hook-related APIs, now it will fallback to `3` if not specified and will be removed in the future
+- core: add a unique message id for each webhook request
+- core: add new APIs for webhook management
+  - `GET /api/hooks/:id/recent-logs` to retrieve recent execution logs(24h) of a webhook
```
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply .changeset/pink-eggs-divide.md:27]: Currently, we don't. Webhook logs are now part of audit logs; we need a new project to track the iteration of the audit logs.

[the remark this answers is not in the document]
Comment (gao-sun) [reply .changeset/pink-eggs-divide.md:27]: @simeng-li can you double check this?

File: .changeset/pink-eggs-divide.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+"@logto/integration-tests": minor
+"@logto/schemas": minor
+---
+
+Provide new features for webhooks
+
+## Features
+
+- Manage webhooks via the Admin Console
+- Securing webhooks
```
Comment (gao-sun) [inline .changeset/pink-eggs-divide.md:13]:
```suggestion
- Securing webhooks by validating signature
```

Comment (xiaoyijun) [reply .changeset/pink-eggs-divide.md]: updated

**Review by gao-sun — APPROVED** [review]

**Pull request merged**